README.md

```diff
@@ -35,6 +35,44 @@ or, if you want to start the development environment
 npm run dev
 ```
 
+## Running [rclone](https://rclone.org) against Permanent.org instances
+
+- Install rclone, using [official install guide](https://rclone.org/install/)
+
+### Setting up an rclone remote
+
+- `rclone config`
+- Select option n for new and enter a remote name such as `permanent-prod` or `permanent-dev`.
+- Set `SFTP` as storage protocol by by selecting from displayed menu, after name prompt.
+- Set an `ssh` host, the host format is `sftp.${environment}.permanent.org` for example `sftp.dev.permanent.org` for `dev` and `staging`. The prod environment is an exception and simply has host as `sftp.permanent.org`.
+- Set you permanent username once prompted for example `rlcone.joy@gmail.com`.
```

Comment:
It's not a big deal, but having a misspelling of rclone in this example email is a bit confusing. It's also better practice to use example.com for these, I think, since this might be a real email address.